niceman/distributions/debian.py

```diff
@@ -543,35 +529,32 @@ def proc_apt_cache_show_cmd(subqueries, cmd_results):
                     p[f] = r[f]
 
     def _get_pkgs_install_date(self, pkg_dicts):
-        # Define a function to process batches of packages and return
-        # a dict from dpkg list filename to creation date
-        def proc_stat_cmd(subqueries, cmd_results):
-            try:
-                out, _ = self._session.execute_command(
-                    ['stat', '-c', '%n: %Y'] + subqueries
-                )
-            except CommandError as exc:  # file not found
+        # Convert package names to dpkg list filenames
+        queries = []
+        for p in pkg_dicts:
+            queries.append(self._pkg_name_to_dpkg_list_file(p["name"]))
```

Comment:
use list (or generator) comprehension please

Reply:
Yeah, I have a couple of these.  Thanks!